lib/oidc/mixin/index.ts

```diff
@@ -285,16 +285,23 @@ export function mixinOAuth
       options = Object.assign({}, options);
     
       // postLogoutRedirectUri must be whitelisted in Okta Admin UI
-      var defaultUri = window.location.origin;
-      var currentUri = window.location.href;
-      var postLogoutRedirectUri = options.postLogoutRedirectUri
+      const defaultUri = window.location.origin;
+      const currentUri = window.location.href;
+      // Fix for issue/1410 - allow for no postLogoutRedirectUri to be passed, resulting in /logout default behavior
+      //    "If no Okta session exists, this endpoint has no effect and the browser is redirected immediately to the
+      //    Okta sign-in page or the post_logout_redirect_uri (if specified)."
+      // - https://developer.okta.com/docs/reference/api/oidc/#logout
+      const postLogoutRedirectUri = options.postLogoutRedirectUri === null ? null :
+        (options.postLogoutRedirectUri
         || this.options.postLogoutRedirectUri
-        || defaultUri;
+        || defaultUri);
```

Comment:
Be careful -- `options` vs `this.options`. Is there a difference? Maybe a different variable name would help avoid confusion.